Cargo.toml

```diff
@@ -54,3 +54,6 @@ verify = []
 # Extra utility tools
 # Compile policies
 compiler = []
+
+[dev-dependencies]
+shlex = "1.3.0"
```

Comment:
`shlex` is an optional dependency, so instead of re-installing it under dev dependencies, it's better to make it a non-optional dependency.

Reply:
`shlex` needs to remain both optional and in dev-dependencies:
  - Optional: it's only used by the `repl` feature, so other builds don't need it
  - Dev-dependencies: CI runs `cargo test --no-default-features` which excludes `repl`, but tests still need `shlex` to compile

  Making it non-optional would unnecessarily add the dependency to all builds.

  Alternatively, we could accept the dependency overhead but simplify the code and CI in return?